File: BookmarkWeb.API/Models/Chats/Schemas/Conversation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace BookmarkWeb.API.Models.Chats.Schemas
{
    public class Conversation
    {
        [JsonProperty("conversationId")]
        public string ConversationId { set; get; }
        [JsonProperty("bookmark")]
        public InputBookmark Bookmark { set; get; }

        [JsonProperty("message")]
        public InputMessage Message { set; get; }

        [JsonProperty("reply")]
        public InputMessage Reply { set; get; }

        [JsonProperty("user")]
        public InputUser User { set; get; }

        [JsonProperty("name")]
        public string Name { set; get; }
        [JsonProperty("isExpanded")]
        public bool IsExpanded { set; get; }
    }
}

File: nuxt3-awesome-starter-2/types/conversation.ts
export interface InputMessage {
  id: string
  content: string
  isMy: boolean
  isNote: boolean
  note: string
  isNoted: boolean
  updatedAt: Date | null
}

export interface InputBookmark {
  bookmarkId: string
  bookmarkUrl: string
}

export interface InputUser {
  userName: string
}

export interface Conversation {
  conversationId: string
  bookmark: InputBookmark
  message: InputMessage
  reply: InputMessage | null
  user: InputUser
  name: string
  isExpand: boolean
}
